src/pymatgen/io/abinit/abiobjects.py

```diff
@@ -339,7 +338,14 @@ class DefaultVariable:
 DEFAULT = DefaultVariable()
 
 
-class SpinMode(namedtuple("SpinMode", "mode nsppol nspinor nspden"), AbivarAble, MSONable):  # noqa: PYI024
+class _SpinMode(NamedTuple):
```

Comment:
maybe call this `SpinModeTuple` or `SpinModeNT`

Reply:
ok!